packages/bits/src/lib/popover/popover.component.ts

```diff
@@ -285,6 +285,17 @@ export class PopoverComponent implements OnDestroy, OnInit, OnChanges {
         this.overlayComponent?.getOverlayRef()?.updatePosition();
     }
 
+    /**
+     * Resets the size of the popover based on the overlay's position.
```

Comment:
How about the following since there isn't anything about position in the implementation of this method?

```suggestion
     * Resets the size of the popover.
```